Quadriga/src/main/java/edu/asu/spring/quadriga/web/HomeController.java

```diff
@@ -118,7 +126,13 @@ public String home(Locale locale, Model model, Principal principal)
         model.addAttribute("conceptmsg", messages.getProperty("concept_desc"));
         model.addAttribute("dictmsg", messages.getProperty("dictonary_desc"));
         model.addAttribute("networksmsg", messages.getProperty("network_desc"));
+        
+        
+        List<IProject> recentProjects = new ArrayList<IProject>();
+        recentProjects = projectManager.getRecentProjectList(sUserId);

```

Comment:
merge with line 131. This doesn't make sense, first you instantiate "recentProjects" but then you assign the result of "getRecentProjectList" to the same variable.